.github/workflows/continuous-integration.yaml

```diff
@@ -0,0 +1,26 @@
+name: Continuous Integration
+on:
+  push:
+    branches:
+      - '**'
```

Comment:
Add continuous integration pipeline to automatically execute unit tests after each change.